### DISCUSSION TITLE
Status of MeshCentral development?

### DISCUSSION THREAD

**Discussion by PathfinderNetworks** [Ylianst/MeshCentral#4945] [General]
Just wondering if Ylian or Bryan (or anyone else) has any updates as t0 the current and future development of MeshCentral?  I understand Ylian and his team are (or at least were- not sure of current status) looking at their options for moving forward after the Intel announcement.  
It's been a couple of months since then so was just wondering if there is any news?  I, and I am certain a great number of others, don't want to see MeshCentral become a stagnant project. But I also understand that Ylian, Bryan, and whoever else was impacted need to make the best decision possible for themselves.

**Comment by The-Real-Admin-Guy** [comment]
There is a new update available from 1.1.2 to 1.1.4  
I do not know what changed but its installable right now.  I guess that settles the development being is still on.

I cant find documentation on the update, must be minor.

**Reply by dinger1986** [reply]
you can find release notes here https://github.com/Ylianst/MeshCentral/releases

**Reply by The-Real-Admin-Guy** [reply]
Muchos Gracias!
